OData/src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -267,6 +279,7 @@ private Expression BindAccessor(SingleValueNode node)
                 case QueryNodeKind.SingleValueOpenPropertyAccess:
                     var openNode = node as SingleValueOpenPropertyAccessNode;
                     return Expression.Property(BindAccessor(openNode.Source), openNode.Name);
+                case QueryNodeKind.None:
```

Comment:
Is this intended to catch the CountVirtualPropertyNode since its Kind is None?  Won't navNode be null since its not a SingleNavigationNode and crash when accessing a property, i.e. navNode.Source?